### PR TITLE
Adding Roci.fi

### PR DESCRIPTION
**Pull request by pavel-filippov-lw**
Adding RociFi to the DefiLlama
https://rocifi.atlassian.net/browse/ROCI-726
https://docs.llama.fi/list-your-project/readme

### CODE REVIEW

File: projects/rocifiv1/index.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        "internalType": "uint256",
+        "name": "",
+        "type": "uint256"
+      }
+    ],
+    "stateMutability": "view",
+    "type": "function"
+};
+
+async function tvl(timestamp, block, chainBlocks) {
```
Comment (OxideDall) [inline projects/rocifiv1/index.js:37]: Can be simplified to `function` that returns `Promise.all([balanceUSDC, balanceWETH]) : Promise<{`polygon:${USDC}`: balanceUSDC,  `polygon:${WETH}`: balanceWETH}>` but if there is no strict requirements for speed, then looks good.

**Review by OxideDall — APPROVED** [review]
LGTM in general

File: projects/rocifiv1/index.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+    const usdcBalance = (await sdk.api.abi.multiCall({
+        calls: ROCI_POOLS.map((pool) => ({
+            target: ROCI_REVENUE_MANAGER,
+            params: pool
+        })),
+        abi: RociRevenueManagerABI,
+        chain: 'polygon',
+        block: chainBlocks.polygon
+    })).output.reduce((a, b) => Number(a) + Number(b.output), 0);
```
Comment (ksamarin90) [inline projects/rocifiv1/index.js]: Is it fine to use Number? Aren't big numbers involved?

**Review by ksamarin90 — APPROVED** [review]

File: projects/rocifiv1/index.js (hunk 2 above)
Comment (pavel-filippov-lw) [reply projects/rocifiv1/index.js]: You're right, BN should be safer for reducing

File: projects/rocifiv1/index.js (hunk 1 above)
Comment (pavel-filippov-lw) [reply projects/rocifiv1/index.js:37]: Good idea. Added.

File: projects/rocifiv1/index.js [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+        })),
+        abi: RociRevenueManagerABI,
+        chain: 'polygon',
+        block: chainBlocks.polygon
+    }));
+
+    await Promise.all([wethBalance, usdcBalance]).then((values) => {
+        console.log(values)
+        balances[`polygon:${WETH}`] = values[0].output;    
+        balances[`polygon:${USDC}`] = values[1].output.reduce((a, b) => new BigNumber(a).plus(new BigNumber(b.output)), 0).toFixed(0);
```
Comment (ksamarin90) [inline projects/rocifiv1/index.js:61]: `a` is already BigNumber

Comment (ksamarin90) [inline projects/rocifiv1/index.js]:
```suggestion
        balances[`polygon:${USDC}`] = values[1].output.reduce((a, b) => a.plus(new BigNumber(b.output)), new BigNumber(0)).toFixed(0);
```

**Review by ksamarin90 — APPROVED** [review]

**Pull request merged**